ubr/psql_target.py

```diff
@@ -166,19 +166,34 @@ def backup(path_list, destination=None):
         'output': [_backup(dbname, destination) for dbname in path_list if dbexists(dbname)]
     }
 
-def _restore(dbname, backup_dir):
+def backup_missing_prompt_user(dbname, dump_path):
```

Comment:
in sometime-interactive tools it is standard to include a `--no-interaction` cli flag for usage from Salt formulas and similar headless environments. If that flag is set, this function should throw an exception; that way we avoid UBR processes being accidentally stuck for hours in a cron job waiting for input